content/en/network_monitoring/cloud_network_monitoring/guide/detecting_application_availability.md

```diff
@@ -0,0 +1,125 @@
+
+### Service-to-Service dependency tracking
+
+CNM tracks dependencies between services, which is essential for ensuring system performance. It helps verify important connections and highlights traffic volumes, ensuring all critical dependencies are operational.
+
+For example, a possible cause of service latency could be too much traffic being directed to a destination endpoint, overwhelming its ability to handle incoming requests effectively.
+
+To analyze the cause of service latency, use the following steps:
+
+1. On the [Network Analytics][1] page, aggregate traffic by `service`, and filter for the cloud region where you may be noticing alerts or service latency. This view displays all service-to-service dependency paths within that region. 
+
+2. Sort the dependency table based on retransmits or latency, to identify connections with the most significant performance degradation. For instance, if you notice an unusually high number of TCP established connections alongside spikes in retransmits and latency, it may indicate that the source is overwhelming the destination's infrastructure with requests.
+
+   {{< img src="network_performance_monitoring/guide/detecting_network_insights/cnm_service_region_retransmits.png" alt="CNM analytics page, grouping by service and region for specific cloud region">}}
+
+3. Click into one of the traffic paths on this page to open the side panel. This provides more detailed telemetry to help you further debug your network dependencies. 
+
+4. While on the side panel view, check the Flows tab to determine if the communication protocol is TCP or UDP, and review metrics like RTT, Jitter, and packets sent and received to help identify the cause of a high retransmit count.
+
+   {{< img src="network_performance_monitoring/guide/detecting_network_insights/cnm_sidepanel_flows.png" alt="Side panel of a traffic flow, highlighting the Flows tab">}}
+
+## Visibility into network traffic 
+
+Datadog CNM consolidates relevant distributed traces, logs, and infrastructure data into a single view, allowing you to identify and trace issues back to the originating request from an application.
+
+In the example below, check the Traces tab under Network Analytics to view distributed traces of requests between source and destination endpoints, which can help you pinpoint where application-level errors occur.
+
+To identify if an issue is an Application or Network issue, you can use the following steps:
+
+1. Navigate to [Infrastructure > Cloud Network > Analytics][1].  
+2. In the Summary graphs, click a line of communication that has a lot of volume and high RTT time:
+
+   {{< img src="network_performance_monitoring/guide/detecting_network_insights/cnm_isolate_series.png" alt="CNM analytics page, clicking on a path with high RTT Time">}}
+
+3. Click **Isolate this series**  
+4. Click into this path and click on the Flows tab to observe RTT time:
```

Comment:
I'm not sure what "this path" is - the line of communication from step 2?
```suggestion
4. Click this path, then click the **Flows** tab to observe RTT time:
```

Reply:
I think this opens a page or a side panel? Future steps talk about "this page," and I think to clarify expectations, it might be worth describing what happens as a result of clicking the path if it opens something new.